src/core/userDataHelper.js

```diff
@@ -30,6 +33,35 @@ var userDataHelper = {
       return userId === team.userid;
     });
   },
+  /**
+   * Get the list of tagged words for a user's notes. Memoize result.
+   * @param userid
+   * @param notes
+   * @returns {Array}
+   */
+  getWordbankWords: function(userid, notes) {
```

Comment:
yeah exactly. This way there are several options as how we initial seed the wordbank. 
- hardcoded seed list
- configured seed list via api
- user generated list via api + configured list